### ISSUE TITLE
Request to have a shell version of the script

### ISSUE THREAD

**Issue by mrhovunping** [vishnubob/wait-for-it#121]
Hi team, 

The script is quite useful, we used it a lot on linux container (ubuntu based). We have switched to Alpine version container and script does not work on this version because "bash" is not the default shell. While there is a work around to install bash on alpine, it would be nice to have a sh version of the script.

**Comment by lwblackledge** [comment]
I found a shell version here: https://gist.github.com/yahyaergun/597d787c6aeffd495e9eaa8b3fdd8e96
A similar issue also linked this go-powered binary version: https://github.com/roerohan/wait-for-it

**Comment by roerohan** [comment]
> A similar issue also linked this go-powered binary version: https://github.com/roerohan/wait-for-it

Yep, I had the same issue. I was using the `scratch` container, which doesn't have `bash` in it (which is why I re-wrote it in go, to package it into a binary executable). It's not been updated for a while, but ideally, it should run on alpine.

**Comment by atkrad** [comment]
Hey @mrhovunping ,

You can install the [Wait4X](https://github.com/atkrad/wait4x) from the official Alpine repositories:
`apk add wait4x`

**Comment by alexislefebvre** [comment]
Duplicate of #18? 
- #18
